stories/Containers/Footnote.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { Footnote } from '../../src';
+
+storiesOf('Footnote', module)
+    .addDecorator(withKnobs)
+    .add('with default', () => (
+        <Footnote show>
```

Comment:
- Make show a knob to test the animation
 - Make the text also a knob for testing :p